src/clanhr_api/core.clj

```diff
@@ -27,9 +27,17 @@
                             (:http-opts opts))}
          opts))
 
+(defn throws-an-error-about-request-time
```

Comment:
I'd change the name, because this actually doesn't throw an exception, it just logs it to bugsnag.